apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -42,7 +43,7 @@ public sealed class SpannerCommand : DbCommand
         public SpannerCommand()
         {
             DesignTimeVisible = true;
-            _commandTimeout = ConnectionPoolOptions.Instance.TimeoutMilliseconds;
+            _commandTimeout = (int)ConnectionPoolOptions.Instance.Timeout.TotalSeconds;
```

Comment:
Another TimeSpan? The fact that this has changed unit from seconds to milliseconds (without changing name) is justification for avoiding such ambiguity, IMO :)